RestaurantApp/Pages/MasterPage.xaml

```diff
@@ -0,0 +1,34 @@
+﻿<ContentPage
+                  xmlns="http://xamarin.com/schemas/2014/forms"
+                  xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                  x:Class="RestaurantApp.Pages.MasterPage"
+                  xmlns:local="clr-namespace:RestaurantApp.Models;assembly=RestaurantApp"
+                  xmlns:pages="clr-namespace:RestaurantApp.Pages;assembly=RestaurantApp"
+                  Title="Menu">
+
+        <ListView
+            x:Name="listView"
+            x:FieldModifier="public"
+            SeparatorVisibility="None">
+           <ListView.ItemsSource>
+                <x:Array Type="{x:Type local:MasterPageItem}">
+                    <local:MasterPageItem Title="Main menu" IconSource="icon.png" TargetType="{x:Type pages:MenuPage}" />
```

Comment:
that's a way, but according to xamarin convention: no data in xaml. so better to bind it from VM (view model)